dune_client/query.py

```diff
@@ -12,8 +12,8 @@
 class Query:
     """Basic data structure constituting a Dune Analytics Query."""
 
-    name: str
     query_id: int
+    name: Optional[str] = "unnamed"
```

Comment:
Also liked you've added the name. Going to be using this in the excel export!

Reply:
The name was always here, but this just makes it optional with a default value so that it's less annoying when you don't want to use it.